pkg/oomstore/revision.go

```diff
@@ -24,6 +25,46 @@ func (s *OomStore) GetRevisionBy(ctx context.Context, groupID int, revision int6
 	return s.metadata.GetRevisionBy(ctx, groupID, revision)
 }
 
+func (s *OomStore) CreateRevision(ctx context.Context, opt metadata.CreateRevisionOpt) (int, string, error) {
+	var revisionID int
+	var snapshotTable string
+	var dummyRevision *types.Revision
+
+	if err := s.metadata.WithTransaction(ctx, func(c context.Context, db metadata.DBStore) error {
+		_, err := s.metadata.GetRevisionBy(ctx, opt.GroupID, 0)
+		if err != nil {
+			if !errdefs.IsNotFound(err) {
+				return err
+			}
+
+			if _, _, err = s.metadata.CreateRevision(ctx, metadata.CreateRevisionOpt{
+				Revision:    0,
+				GroupID:     opt.GroupID,
+				Description: "dummy revision it is useful for join and export",
```

Comment:
suggestion: dummy revision will be used at Join and Export

Reply:
done